grind/bin/grind

```diff
@@ -355,6 +355,10 @@ class TestRunner:
                             type=int,
                             default=default_timeout,
                             help="Per-task timeout. The default timeout is %s seconds." % default_timeout)
+        parser.add_argument('--java-version',
+                            type=int,
+                            default=7,
```

Comment:
This might be nice to set in pconfig rather than as a new flag, since the JDK version is normally fixed for a repo. That way you don't need to specify it each time.